haigha/transports/socket_transport.py

```diff
@@ -29,26 +34,33 @@ def connect(self, (host, port), klass=socket.socket):
         Connect assuming a host and port tuple.
         '''
```

Comment:
Need to document exceptions that this method may reasonably raise, so callers know what to expect